### PR TITLE
Require essential packages for tmt testing

### PR DESCRIPTION
**Pull request by psss**
As for now tests would fail unless tmt packages were installed on
the guest manually or explicitly in the prepare step.

### CODE REVIEW

**Comment by lukaszachy** [comment]
Hover I'm not sure we want this. With this change I have to have installed tmt packages - pip install isn't enough.

File: tests/libraries/local/main.fmf [hunk 1]
```diff
@@ -4,5 +4,7 @@ description:
     works as expected. Fetches the example library from upstream,
     modifies it a bit, references it from a test and verifies the
     changed functionality by running the test in a container.
-require: git
+require+:
+  - git
```
[the remark this answers is not in the document]
Comment (psss) [reply tests/libraries/local/main.fmf:8]: Good point. And `git-core` is already required by `tmt`. Removing.

**Comment by psss** [comment]
> Hover I'm not sure we want this. With this change I have to have installed tmt packages - pip install isn't enough.

Hmmm, interesting point. Do you find this as an important issue? Seems we can't have both pure-pip and auto-rpm or how to call it. I ran into the problem when trying:

    tmt test export --fmf-id | wow --fmf-id -

Or when running tests using a different provision method:

    tmt run -a provision -h virtual

All tests failed because there was no `tmt` installed on the guest. Which is because we don't have it in the `require` key. I find it ok to require `tmt` installed on the guest (laptop) as it does not prevent user from testing from venv as well.

**Comment by lukaszachy** [comment]
> All tests failed because there was no `tmt` installed on the guest. Which is because we don't have it in the `require` key. I find it ok to require `tmt` installed on the guest (laptop) as it does not prevent user from testing from venv as well.

In this case you have rpm installed just to make rpm command happy. You are testing different bits than you request to have installed. But if this makes life easier for new developers then whatever.

**Comment by psss** [comment]
> In this case you have rpm installed just to make rpm command happy.

Yeah, not a completely pure/clean solution, but I believe it's better than keeping the `require` empty. Do you see any nice workaround for the `tmt test export | wow` use case?

**Comment by lukaszachy** [comment]
> Do you see any nice workaround for the `tmt test export | wow` use case?

I'm afraid I can't see the problem there. Why it needs workaround?

**Review by lukaszachy — APPROVED** [review]

**Pull request merged**